content/intro-to-storybook/svelte/en/simple-component.md

```diff
@@ -92,46 +71,61 @@ import { actions, task } from './storybook-helper';
 
 export default {
   title: 'Task',
+  excludeStories: /.*Data$/,
+};
+
+export const actionsData = {
+  onPinTask: action('onPinTask'),
+  onArchiveTask: action('onArchiveTask'),
+};
+
+export const taskData = {
+  id: '1',
+  title: 'Test Task',
+  state: 'Task_INBOX',
+  updated_at: new Date(2019, 0, 1, 9, 0),
 };
 
 // default task state
-export const standard = () => ({
+export const Default = () => ({
   Component: Task,
   props: {
-    task,
+    task: taskData,
   },
   on: {
-    ...actions,
+    ...actionsData,
   },
 });
 // pinned task state
-export const pinned = () => ({
+export const Pinned = () => ({
   Component: Task,
   props: {
     task: {
-      ...task,
+      ...taskData,
       state: 'TASK_PINNED',
     },
   },
   on: {
-    ...actions,
+    ...actionsData,
   },
 });
 // archived task state
-export const archived = () => ({
+export const Archived = () => ({
   Component: Task,
   props: {
     task: {
-      ...task,
+      ...taskData,
       state: 'TASK_ARCHIVED',
     },
   },
   on: {
-    ...actions,
+    ...actionsData,
   },
 });
 ```
 
+Due to some language restraints the default action was named `Default` as the lowercase it's a reserved word, the `excludeStories` prop was added to avoid Storybook treating both the `taskData` and `actionsData` as actual stories, you can read more about it [here](https://storybook.js.org/docs/formats/component-story-format/#non-story-exports).
```

Comment:
> Due to some language restraints the default action was named `Default`

We are moving (SB-wide) to using uppercase names for story exports, so I don't think we need to make excuses for it, I would say let's just call it `Default` and not really say why, that's just what we call stories.

Reply:
@tmeasday see my latest commit. Simplified it a bit.

Reply:
LGTM